### ISSUE TITLE
False-positive PIE796: Enum contains duplicate value: `enum.auto()`

### ISSUE THREAD

**Issue by actionless** [charliermarsh/ruff#1932]
ruff 0.0.224

```python
import enum


class PackageSource(enum.Enum):
    REPO = enum.auto()
    AUR = enum.auto()  # E: PIE796 Enum contains duplicate value: `enum.auto()`
    LOCAL = enum.auto()  # E: PIE796 Enum contains duplicate value: `enum.auto()`
```

**Comment by charliermarsh** [comment]
Ah right, totally. Will fix today.
